spec/en/reference.md

```diff
@@ -43,6 +44,7 @@ This specification includes the following files along with their associated cont
 |  [transfers.txt](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#transferstxt)  | Optional | Rules for making connections at transfer points between routes. |
 |  [feed_info.txt](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#feed_infotxt)  | Optional | Additional information about the feed itself, including publisher, version, and expiration information. |
 |  [*__board_alight.txt__*](#board_alighttxt) | Optional | Tracks boardings/alightings along with associated information at stop-level. |
+|  [*__trip_capacity.txt__*](#trip_capacitytxt) | Optional | Identifies the capacities of a vehicle operating a trip on a given day. |
```

Comment:
If we want it to be for a given trip on a given day, should we not have those be required fields in the file?